lib/galaxy/tools/deps/conda_util.py

```diff
@@ -414,9 +461,9 @@ def install_conda_targets(conda_targets, env_name=None, conda_context=None):
         ]
         for conda_target in conda_targets:
             create_args.append(conda_target.package_specifier)
-        return conda_context.exec_create(create_args)
+        return conda_context.exec_create(create_args, allow_local=allow_local)
     else:
-        return conda_context.exec_install([t.package_specifier for t in conda_targets])
+        return conda_context.exec_install([t.package_specifier for t in conda_targets], allow_local=allow_local)
 
 
 def install_conda_target(conda_target, conda_context=None, skip_environment=False):
```

Comment:
Seems premature I guess - I think there is only one scenario where it needs to be forbidden - when use_local is enabled and ``conda-build`` is being installed with ``install_conda_targets``.